### PR TITLE
Add GetAuthorities method

### PR DESCRIPTION
**Pull request by fgimenez**
This will allow feeding versionBundles info into clusterconfig chart.

### CODE REVIEW

File: pkg/framework/version_bundle.go [hunk 1]
```diff
@@ -46,6 +46,25 @@ func GetVersionBundleVersion(params *VBVParams) (string, error) {
 	return output, nil
 }
 
+func GetAuthorities(params *VBVParams) ([]versionbundle.Authority, error) {
+	err := checkType(params.VType)
+	if err != nil {
+		return []versionbundle.Authority{}, microerror.Mask(err)
```
Comment (xh3b4sd) [inline pkg/framework/version_bundle.go]: Zero value of a slice is `nil`.

File: pkg/framework/version_bundle.go [hunk 2]
```diff
@@ -46,6 +46,25 @@ func GetVersionBundleVersion(params *VBVParams) (string, error) {
...
+	}
+	log.Printf("Tested version %q", params.VType)
```
Comment (xh3b4sd) [inline pkg/framework/version_bundle.go]: I would like to use our logger as we do everywhere. This here will not play in favour of our perception when we humans deal with logs.

**Review by tuommaki — APPROVED** [review]
As Tim suggested, I'd also prefer `nil` instead of empty slice when returning error. Otherwise LGTM.

File: pkg/framework/version_bundle.go (hunk 1 above)
Comment (fgimenez) [reply pkg/framework/version_bundle.go]: Sure, done.

File: pkg/framework/version_bundle.go (hunk 2 above)
Comment (fgimenez) [reply pkg/framework/version_bundle.go]: Done, PTAL.

**Review by xh3b4sd — APPROVED** [review]

**Pull request merged**